src/WatchDogs.Infrastructure.DxTrade/ISessionTokenStorage.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.DxTrade
+{
```

Comment:
I would suggest switching to file-scoped namespaces because they are reducing nesting (curiously `DxTradeConnectionOptions` is a file-scoped so see how it looks). Go to VS -> Tools -> Options -> Text Editor -> C# -> Code Style -> General -> Code Block Preferences -> Namespace declarations -> File Scoped

Reply:
Btw don't change namespace styles in these classes, just change your settings.

Reply:
Done.